OSM data program V2.ipynb

```diff
@@ -0,0 +1,1086 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9425e625",
+   "metadata": {},
+   "source": [
+    "## Importing Packages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "27a9178e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pyrosm import get_data, OSM\n",
+    "from pyrosm.data import sources\n",
+    "import os\n",
+    "import pandas as pd\n",
+    "from sqlalchemy import create_engine"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cb1cf4f1",
+   "metadata": {},
+   "source": [
+    "### Functions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "b0013bf6",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "def get_location(location, folder, directory):\n",
```

Comment:
The arguments in the function should have typing annotation since python is not checking the typing.

Reply:
also the returning types.